packages/amplify-cli/src/commands/delete.js

```diff
@@ -1,6 +1,12 @@
 module.exports = {
   name: 'delete',
   run: async context => {
+    if ((context.parameters.array || []).length > 0) {
+      context.print.error('"delete" command does not expect additional arguments.');
+      context.print.info('Perhaps you meant to use the "remove" command instead of "delete"?');
```

Comment:
nit: I think having both console prints be cataloged as an error (have both display in red color) would be better for the user.

Reply:
I agree with that, good idea 👍 Let me make this change...